include/onnxruntime/core/providers/tensorrt/tensorrt_provider_options.h

```diff
@@ -5,8 +5,8 @@
 
 /// <summary>
 /// Options for the TensorRT provider that are passed to SessionOptionsAppendExecutionProvider_TensorRT_V2.
-/// Please note that this sturct is identical to OrtTensorRTProviderOptions but only to be used internally. 
-//  User can only get the instance of OrtTensorRTProviderOptionsV2 via CreateTensorRTProviderOptions.
+/// Please note that this struct is identical to OrtTensorRTProviderOptions but only to be used internally. 
```

Comment:
CC: @chilo-ms. Awkward to see this file in platform. So, moved it to providers instead.